syntax = "proto3";

import "hq/dto/Security.proto";
import "hq/dto/OrderBook.proto";
option java_package = "com.huasheng.quant.open.sdk.protobuf.hq.notify";
option java_outer_classname = "OrderBookFullNotifyProto";
message OrderBookFullNotify {
    Security security = 1;
    int32 side = 2; //0是买，1是卖
    repeated OrderBook orderBookList = 3; // 买卖档列表
    int32 mktTmType = 4;
}